test/Benchmarks/Transactions/TransactionBenchmark.cs

```diff
@@ -99,7 +100,8 @@ public sealed class SiloTransactionConfigurator : ISiloBuilderConfigurator
         {
             public void Configure(ISiloHostBuilder hostBuilder)
             {
-                hostBuilder.UseDistributedTM();
+                hostBuilder.UseDistributedTM()
```

Comment:
Woah... we seriously need to change this extension method name ASAP

Opened #4688 about it.